diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/IndentationRule.kt

```diff
@@ -238,15 +248,61 @@ class IndentationRule(configRules: List<RulesConfig>) : DiktatRule(
                 .trim())
     }
 
+    /**
+     * This method fixes all lines of string template except the last one
+     * Also it considers $foo insertions in string
+     */
+    private fun fixFirstTemplateEntries(
+        node: ASTNode,
+        textIndent: String,
+        actualIndent: Int
+    ) {
+        val correctedText = StringBuilder()
+        // shift of the node depending on its initial string template indent
+        val nodeStartIndent = if (node.firstChildNode.text.takeWhile { it == ' ' }.count() - actualIndent - INDENT_SIZE > 0) {
+            node.firstChildNode.text.takeWhile { it == ' ' }.count() - actualIndent - INDENT_SIZE
+        }
+        else {
+            0
+        }
+
+        when {
+            // if string template is before literal_string
+            node.treePrev.elementType in stringLiteralTokens && node.treeNext.elementType !in stringLiteralTokens -> {
+                correctedText.append(node.firstChildNode.text.trimEnd())
+            }
+            // if string template is after literal_string
+            node.treePrev.elementType !in stringLiteralTokens && node.treeNext.elementType in stringLiteralTokens -> {
```

Comment:
Please create two expressions `node.treePrev.elementType in stringLiteralTokens` and `node.treeNext.elementType in stringLiteralTokens`. And unify this logic